src/System.Diagnostics.Process/tests/ProcessTests.Linux.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Security;
+using Xunit;
+using Xunit.NetCore.Extensions;
+
+namespace System.Diagnostics.Tests
+{
+    public partial class ProcessTests : ProcessTestBase
+    {
+        [Fact]
```

Comment:
please make tests that pop up windows (like shell exec ones) be [OuterLoop] to not be obnoxious. Note CI doesn't normally run outer loop, so if you want to use CI for these tests, you'll either have to request outer loop or change them to outer loop in your last commit.

Reply:
I added (and for now commented out) [OuterLoop]  for these tests. On the final commit I will uncomment those lines.